fe/fe-core/src/main/java/org/apache/doris/tablefunction/MetadataGenerator.java

```diff
@@ -574,6 +537,36 @@ private static TFetchSchemaTableDataResult queriesMetadataResult(TSchemaTableReq
                 trow.addToColumnValue(new TCell().setStringVal(""));
             }
             trow.addToColumnValue(new TCell().setStringVal(selfNode));
+
+            QueueToken queueToken = queryInfo.getQueueToken();
```

Comment:
Do not use token directly here.
Add related method in QueryInfo, for example getQueueStartTime, getQueueEndTime
and then QueryInfo could use coordinator to get these time.